### ISSUE TITLE
Unable to install Terraform

### ISSUE THREAD

**Issue by JelleSmet-TomTom** [dflook/terraform-github-actions#188]
### Problem description

For some reason the `Installing Terraform` step fails with `urllib.error.HTTPError: HTTP Error 404: Not Found`

### Backend

Azure blob storage

### Workflow YAML

```yaml
- name: Terraform Apply
      if: inputs.action == 'apply'
      uses: dflook/terraform-apply@v1
      id: terraform-apply
      with:
        path: .github/_scratch/xxxxxxxxxxxxxxxxxxxxxxx
        var_file: |-
          ${{ inputs.solution_varfiles }}
          .github/_scratch/secrets.hcl
        variables: |-
          ${{ inputs.solution_variables }}
        backend_config_file: ${{ inputs.solution_backend_config_file }}
        auto_approve: ${{ inputs.auto_approve == 'true' }}
      env:
        GITHUB_TOKEN: ${{ inputs.github_token }}
        TERRAFORM_HTTP_CREDENTIALS: |
          github.com/xxxxxxxxxxxxx=oauth2:${{ inputs.xxxxxxxxxxx }}
        ARM_CLIENT_ID: ${{ inputs.xxxxxxxxxxxx }}
        ARM_CLIENT_SECRET: ${{ inputs.xxxxxxxxxx }}
        ARM_SUBSCRIPTION_ID: ${{ inputs.xxxxxxxxxxx }}
        ARM_TENANT_ID: ${{ inputs.target_tenant }}

        TERRAFORM_PRE_RUN: |
          apt-get update
          apt-get install -y azure-cli
          curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash -s -- --version v3.6.3
          curl -L -o kubelogin.zip https://github.com/Azure/kubelogin/releases/download/v0.0.9/kubelogin-linux-amd64.zip
          unzip kubelogin.zip && rm kubelogin.zip
          [ ! -f /usr/local/bin/kubelogin ] && cp -v bin/linux_amd64/kubelogin /usr/local/bin/kubelogin && chmod -v 0755 /usr/local/bin/kubelogin
```

### Workflow log

```shell
2022-05-09T21:29:06.6303675Z   TERRAFORM_PRE_RUN: apt-get update
apt-get install -y azure-cli
curl -L -o kubelogin.zip https://github.com/Azure/kubelogin/releases/download/v0.0.9/kubelogin-linux-amd64.zip
unzip kubelogin.zip && rm kubelogin.zip
[ ! -f /usr/local/bin/kubelogin ] && cp -v bin/linux_amd64/kubelogin /usr/local/bin/kubelogin && chmod -v 0755 /usr/local/bin/kubelogin

2022-05-09T21:29:06.6304374Z ##[endgroup]
2022-05-09T21:29:06.6499548Z ##[command]/usr/bin/docker run --name danielflookterraformgithubactionssha25632ae948030e86c4a1d88709773922ed818bf3ccee0134a1b86fa269a07d22d39_faf661 --label 08450d --workdir /github/workspace --rm -e GITHUB_TOKEN -e TERRAFORM_HTTP_CREDENTIALS -e ARM_CLIENT_ID -e ARM_CLIENT_SECRET -e ARM_SUBSCRIPTION_ID -e ARM_TENANT_ID -e TERRAFORM_PRE_RUN -e INPUT_PATH -e INPUT_VAR_FILE -e INPUT_VARIABLES -e INPUT_BACKEND_CONFIG_FILE -e INPUT_ADD_GITHUB_COMMENT -e INPUT_WORKSPACE -e INPUT_BACKEND_CONFIG -e INPUT_VAR -e INPUT_PARALLELISM -e INPUT_TARGET -e INPUT_REPLACE -e INPUT_LABEL -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_RETENTION_DAYS -e GITHUB_RUN_ATTEMPT -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_REF_NAME -e GITHUB_REF_PROTECTED -e GITHUB_REF_TYPE -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e GITHUB_ACTION_REPOSITORY -e GITHUB_ACTION_REF -e GITHUB_PATH -e GITHUB_ENV -e GITHUB_STEP_SUMMARY -e GITHUB_ACTION_PATH -e RUNNER_OS -e RUNNER_ARCH -e RUNNER_NAME -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true --entrypoint "/entrypoints/plan.sh" -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/cloudplatform-kubernetes-observability/xxxxxxxxxxxxxxxxxxx":"/github/workspace" danielflook/terraform-github-actions@sha256:32ae948030e86c4a1d88709773922ed818bf3ccee0134a1b86fa269a07d22d39
2022-05-09T21:29:07.2201554Z ##[group]Installing Terraform
2022-05-09T21:29:09.6632763Z Traceback (most recent call last):
2022-05-09T21:29:09.6633237Z Terraform version not specified, using the latest version
2022-05-09T21:29:09.6636270Z   File "/usr/local/bin/terraform-version", line 8, in <module>
2022-05-09T21:29:09.6636650Z Switching to Terraform vNone
2022-05-09T21:29:09.6637520Z     sys.exit(main())
2022-05-09T21:29:09.6640021Z   File "/usr/local/lib/python3.9/dist-packages/terraform_version/__main__.py", line 119, in main
2022-05-09T21:29:09.6640813Z     switch(version)
2022-05-09T21:29:09.6641710Z   File "/usr/local/lib/python3.9/dist-packages/terraform_version/__main__.py", line 97, in switch
2022-05-09T21:29:09.6642159Z     target_path = get_executable(version)
2022-05-09T21:29:09.6642740Z   File "/usr/local/lib/python3.9/dist-packages/terraform/download.py", line 104, in get_executable
2022-05-09T21:29:09.6643195Z     return download_version(version, download_dir)
2022-05-09T21:29:09.6643801Z   File "/usr/local/lib/python3.9/dist-packages/terraform/download.py", line 67, in download_version
2022-05-09T21:29:09.6644240Z     local_filename, headers = urlretrieve(
2022-05-09T21:29:09.6644637Z   File "/usr/lib/python3.9/urllib/request.py", line 239, in urlretrieve
2022-05-09T21:29:09.6645060Z     with contextlib.closing(urlopen(url, data)) as fp:
2022-05-09T21:29:09.6645479Z   File "/usr/lib/python3.9/urllib/request.py", line 214, in urlopen
2022-05-09T21:29:09.6645857Z     return opener.open(url, data, timeout)
2022-05-09T21:29:09.6646264Z   File "/usr/lib/python3.9/urllib/request.py", line 523, in open
2022-05-09T21:29:09.6646643Z     response = meth(req, response)
2022-05-09T21:29:09.6647021Z   File "/usr/lib/python3.9/urllib/request.py", line 632, in http_response
2022-05-09T21:29:09.6647402Z     response = self.parent.error(
2022-05-09T21:29:09.6647773Z   File "/usr/lib/python3.9/urllib/request.py", line 561, in error
2022-05-09T21:29:09.6648122Z     return self._call_chain(*args)
2022-05-09T21:29:09.6648499Z   File "/usr/lib/python3.9/urllib/request.py", line 494, in _call_chain
2022-05-09T21:29:09.6648850Z     result = func(*args)
2022-05-09T21:29:09.6649221Z   File "/usr/lib/python3.9/urllib/request.py", line 641, in http_error_default
2022-05-09T21:29:09.6649645Z     raise HTTPError(req.full_url, code, msg, hdrs, fp)
2022-05-09T21:29:09.6650033Z urllib.error.HTTPError: HTTP Error 404: Not Found
```

**Comment by dflook** [comment]
Hello @JelleSmet-TomTom, I believe this is fixed in v1.25.1

**Comment by JelleSmet-TomTom** [comment]
Hi @dflook tnx for the quick response. I can confirm this solves the problem: 
```
uses: dflook/terraform-plan@v1.25.1
```
